src/main/scala/uk/gov/hmrc/play/binders/QueryBinders.scala

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
ITs used in other projects, there are no tests in play-url-binders for it. maybe it should have tests, but it looks a very lightweight class